.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    if: github.repository == 'python-twitter-tools/twitter'
```

Comment:
This guard prevents it running on forks. It would fail on forks because they don't have [Test]PyPI API tokens saved in the repo secrets.